### ISSUE TITLE
Vs Code stops working untill refreshed

### ISSUE THREAD

**Issue by Usman1395** [microsoft/vscode#95309]
Issue Type: <b>Bug</b>

Hi...
I am facing the vscode stops working issue on vs code from version 1.31 to latest version till now...
But the issue has not been resolved yet.
when I am on like changing or adding classes in navtabs from bootstrap vscode cpu usage is on peek and I cannot use vscode unless I refresh vscode.
I am using live server extension with it.
and preview on chrome also does not load and have to restart all things again.
This issue is very annoying.
this issue  is also present when giving background image in style attribute inline styling.
This issue is aslo present in many data toggle attributes whereever they are and I have to refresh but I mentioned navtabs because these are specific.
anybody can use this code in their editor and try to add classes in anchor tags and then vscode will become slow and its cpu usage will also high.
Please fix the issue

 <ul class="nav nav-tabs border-0 heading-margin" id="myTab" role="tablist">
                <li class="nav-item w-50 primary-colorh">
                    <a class="nav-link bg-active bg-warning bg-primary-borderh bg-primary-color bg-primary-colorhb bg active" id="experience-tab"
                        data-toggle="tab" data-target="#experience" role="tab" aria-controls="experience"
                        aria-selected="true">Experience</a>
                </li>
                <li class="nav-item w-50 primary-colorh">
                    <a class="nav-link bg-white text-black hover lsp1 shadow " id="education-tab" data-toggle="tab"
                        data-target="#education" role="tab" aria-controls="education"
                        aria-selected="false">Education</a>
                </li>
            </ul>

VS Code version: Code 1.44.1 (a9f8623ec050e5f0b44cc8ce8204a1455884749f, 2020-04-11T01:48:12.622Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4300M CPU @ 2.60GHz (4 x 2594)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.69GB (3.32GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (28)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-css-formatter|aes|1.0.1
laravel-goto-view|cod|1.3.2
bracket-pair-colorizer|Coe|1.0.61
dakara-foldplus|dak|1.4.0
jquerysnippets|don|0.0.1
vscode-html-css|ecm|0.2.3
vscode-great-icons|emm|2.1.47
bootstrap-4-cdn-snippet|eve|1.3.0
auto-rename-tag|for|0.1.1
beautify|Hoo|1.5.0
css-compact|jso|1.0.2
tag-rename|kri|0.2.1
vscode-format-context-menu|lac|1.0.4
expand-region|let|0.1.4
revealfileinfolder|Mol|0.0.3
vscode-autoprefixer|mrm|3.0.0
color-highlight|nau|2.3.0
laravel-blade|one|1.20.0
advanced-new-file|pat|1.2.0
material-icon-theme|PKi|4.0.1
vscode-css-peek|pra|3.0.2
vscode-css-navigation|puc|1.9.0
vscode-formatall|rat|0.0.2
LiveServer|rit|5.6.1
vscode-scss-formatter|sib|2.0.0
sass-indented|syl|1.8.2
open-in-browser|tec|2.0.0
highlight-matching-tag|vin|0.9.9

(1 theme extensions excluded)

</details>

**Comment by chrmarti** [comment]
Does this also occur with extensions disabled? (`F1` > `Developer: Reload With Extensions Disabled`)

**Comment by Usman1395** [comment]
> Does this also occur with extensions disabled? (`F1` > `Developer: Reload With Extensions Disabled`)

Hi I just checked it right now
And if I disable all exptesions then it does not happen like vscode is working without problem.
but when I enable the extensions after adding one class it stops working .and needs to be refreshed

**Comment by Usman1395** [comment]
Could you please help me here that what extension could be causing the problem?

**Comment by chrmarti** [comment]
You could disable them one-by-one in the extension viewlet and check if the problem still occurs.

Please then file an issue for the extension causing it. Thanks.

**Comment by Usman1395** [comment]
Sure...
Thank you
I will try it and let you know about the extension that causes the
problem....
Been in this problem for over a year now and could not identify ghe
problem....
Thank You Very Much for pointing out........
Really appreciate....

**Comment by Usman1395** [comment]
![image](https://user-images.githubusercontent.com/48567433/79551952-d0df0700-80b3-11ea-8a3a-fb277b99cff2.png)
Hi ....
I have identified the problem and this is because of this extension
Auto Rename Tag
formulahendry.auto-rename-tag
Jun Han
2,692,772
Repository
Auto rename paired HTML/XML tag
Disable
Uninstall
This extension is enabled globally.

I have also attached the screenshot to help recognize easily what the extension is.
when I enable it and go to add class vscode stops working and after disabling it works pretty fine.
I am in love with vscode already but after disabling this extension it is way more better performing than before.
Thank You Very Much for pointing out the problem ....
